app/repositories/LisaCacheRepository.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package repositories
+
+import play.api.Configuration
+import play.api.mvc.{AnyContent, Request}
+import uk.gov.hmrc.http.SessionKeys
+import uk.gov.hmrc.mongo.cache.{CacheItem, SessionCacheRepository}
+import uk.gov.hmrc.mongo.{CurrentTimestampSupport, MongoComponent}
+
+import java.util.concurrent.TimeUnit
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.duration.Duration
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class LisaCacheRepository @Inject()(
```

Comment:
missing test spec for LisaCacheRepository

Reply:
I dont think adding tests for the LisaCacheRepository would be helpful. 
This class extends `SessionCacheRepository` which I assume is already tested and dosnt have much other functionality to test.

Reply:
True, `SessionCacheRepository` is already tested.

You could add a test for the function `getFullCache` to make sure it behaves as expected.